conf.yaml

```diff
@@ -1376,6 +1377,25 @@ contents:
               -
                 repo:   swiftype
                 path:   docs
+    -
+        title:      Enterprise Search
+        sections:
+          -
+            title:      Workplace Search
+            prefix:     en/enterprisesearch/workplacesearch
+            index:      workplace-search-docs/index.asciidoc
+            private:    1
+            current:    master
+            branches:   [ master ]
+            single:     1
+            tags:       Workplace Search/Reference
+            subject:    Workplace Search
+            sources:
+              -
+                repo:   enterprise-search
+                path:   workplace-search-docs
```

Comment:
Doesn't this also need `chunk: 1`? Without it, I'm seeing [all content on the same page](http://docs_1724.docs-preview.app.elstc.co/guide/en/enterprisesearch/workplacesearch/current/index.html).

Reply:
Or is that on purpose? I'm just now noticing the `single: 1`, which I haven't seen before.

Reply:
Yes--and the "single" option needed to be removed.